samples/UrlHelperSample/CustomUrlHelper.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Mvc.Routing;
+
+namespace UrlHelperSample
```

Comment:
Is anything other than the `namespace` different? Odd it's not showing up as a rename.